pkg/asset/ignition/bootstrap/baremetal/template.go

```diff
@@ -111,14 +222,52 @@ func GetTemplateData(config *baremetal.Platform, networks []types.MachineNetwork
 	templateData.ExternalStaticDNS = config.BootstrapExternalStaticDNS
 	templateData.ExternalMACAddress = config.ExternalMACAddress
 
-	_, externalURLv6 := externalURLs(config.APIVIPs)
+	// If the user has manually set disableVirtualMediaTLS to False in the Provisioning CR, then enable TLS in ironic.
```

Comment:
This is the function that gets the data that is used to _create_ the Provisioning CR (by substituting it into the [template](https://github.com/openshift/installer/blob/master/data/data/manifests/openshift/baremetal-provisioning-config.yaml.template)). So it's too early to read any changes from the user. The file won't exist for them to edit until after `create manifests` (which calls this) is complete. So you won't be able to read any info from the user until `create ignition`, so in the bootstrap ignition asset.

Reply:
I was actually seeing this execute in dev-scripts under 'create cluster'. Before this, dev-scripts calls 'create manifests'.

Reply:
> This is the function that gets the data that is used to create the Provisioning CR 

No? This asset is used to create the bootstrap VM, I don't think it's used that early?